apps/e2e/cypress/e2e/admin/user-info.cy.js

```diff
@@ -0,0 +1,258 @@
+              });
+            });
+          });
+        });
+      });
+
+    // use new test user to submit an application
+    cy.get("@testUser").then((testUser) => {
+      cy.loginBySubject(testUser.sub);
+      cy.getMe().then((testUser) => {
+        cy.get("@publishedTestPoolAdvertisement").then((poolAdvertisement) => {
+          cy.createApplication(testUser.id, poolAdvertisement.id).then(
+            (poolCandidate) => {
+              cy.submitApplication(poolCandidate.id, uniqueTestId.toString())
+                .its("id")
+                .as("poolCandidateId");
+            },
+          );
+        });
+      });
+    });
+
+    // Create a search request for new test user
+    cy.loginByRole("admin");
+    cy.get("@publishedTestPoolAdvertisement").then((poolAdvertisement) => {
+      cy.get("@testClassification").then((classification) => {
+        cy.get("@testDepartment").then((department) => {
+          createSearchRequest({
+            departmentId: department.id,
+            classificationId: classification.id,
+            poolId: poolAdvertisement.id,
+            searchRequestAlias: "testSearchRequest",
+          });
+        });
+      });
+    });
+
+    // Change test users pool candidate status to qualified available
+    cy.get("@poolCandidateId").then((poolCandidateId) => {
+      cy.updatePoolCandidateAsAdmin(poolCandidateId, {
+        status: PoolCandidateStatus.QualifiedAvailable,
+      });
+    });
+  };
+
+  beforeEach(() => {
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "getMePools");
+      aliasQuery(req, "allPools");
+      aliasQuery(req, "GetPoolCandidatesPaginated");
+      aliasQuery(req, "getPoolCandidateSearchRequests");
+      aliasMutation(req, "UpdatePoolCandidateStatus");
+      aliasMutation(req, "createPoolCandidateSearchRequest");
+    });
+
+    // select some dimensions to use for testing
+    cy.getSkills().then((allSkills) => {
+      cy.wrap(allSkills[0]).as("testSkill"); // take the first skill for testing
+    });
+    cy.getDepartments().then((allDepartments) => {
+      cy.wrap(allDepartments[0]).as("testDepartment"); // take the first department for testing
+    });
+    cy.getGenericJobTitles().then((allGenericJobTitles) => {
+      const testGenericJobTitle = allGenericJobTitles[0]; // take the first ID for testing matching
+      cy.wrap(testGenericJobTitle).as("testGenericJobTitle");
+      cy.wrap(testGenericJobTitle.classification).as("testClassification");
+    });
+  });
+
+  it("Should show non-authorized warning page if user has applicant role", () => {
+    cy.loginByRole("applicant");
+    cy.visit("/en/admin");
+
+    cy.findByRole("heading", {
+      name: /Sorry, you are not authorized to view this page./i,
+    })
+      .should("exist")
+      .and("be.visible");
+  });
+
+  it("Should show the pool operator a candidate's user information page on their team", () => {
```

Comment:
The assertions on these tests seem a bit weak to me.  You're making sure the page loads (or doesn't) but it doesn't seem like you're asserting that certain pool candidates are visible or not.  For example, on this test I'd expect that you can see candidates that belong to the pool of the current pool operator but no candidates of other pools.

That type of assertion seems more appropriate (and easier to setup!) in a phpunit test though...  @tristan-orourke?

Reply:
Hmm. I see your point @petertgiles that testing for exactly which pool candidates are returned by the query would be easier to do in a phpunit test. We should write that phpunit test if we haven't yet.

I would still like to see this cypress test - it can be simpler than my original acceptance criteria, but smarter than it is now. I would like it to create an applicant with applications to two different pools, and then a pool operator in one of those pools is able to view the profile without seeing a "GraphQL error".

Reply:
Alright, I simplified the testing by directly visiting the users info page. I stuck with one user throughout the testing:

- First testing viewing the user info page without creating an application. 
- Then tested viewing the user info page after applying to a pool that's owned by the pool operator. 
- Lastly, tested viewing the user info page after user applies to another pool not synced the pool operator.